src/Microsoft.AspNetCore.SignalR.Http/HubRouteBuilder.cs

```diff
@@ -25,11 +25,11 @@ public HubRouteBuilder(SocketRouteBuilder routes)
         public void MapHub<THub>(string path, Action<HttpSocketOptions> socketOptions) where THub : Hub<IClientProxy>
         {
             // find auth attributes
-            var authorizeAttribute = typeof(THub).GetCustomAttribute<AuthorizeAttribute>(inherit: true);
+            var authorizeAttributes = typeof(THub).GetCustomAttributes<AuthorizeAttribute>(inherit: true);
```

Comment:
Looks like it https://github.com/aspnet/Mvc/blob/c27b07ef3f8b07dfe1d1de38a8d7bf2d6a9298f4/src/Microsoft.AspNetCore.Mvc.Core/Internal/AuthorizationApplicationModelProvider.cs#L50
I added `MapHubFindsMultipleAuthAttributesOnDoubleAuthHub`